torch/csrc/inductor/aoti_runner/model_container_runner.cpp

```diff
@@ -5,23 +5,7 @@
 #include <torch/csrc/inductor/aoti_torch/oss_proxy_executor.h>
 #include <torch/csrc/inductor/aoti_torch/tensor_converter.h>
 
-#ifndef _WIN32
-#include <sys/stat.h>
-#else
 #include <filesystem>
```

Comment:
We need to the experimental backport for old versions of GCC in the internal codebase unfortunately.

Reply:
Fixed